Thumbnail/ConsumerThumbnail.php

```diff
@@ -95,10 +95,7 @@ public function generate(MediaProviderInterface $provider, MediaInterface $media
 
             // BC compatibility for missing EventDispatcher
             if (null === $this->dispatcher) {
-                trigger_error(
-                    'Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.',
-                    E_USER_DEPRECATED
-                );
+                trigger_error('Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
@greg0ire I was going to squash all the commits (like requested in the comments from Soullivaneuh) but I wasn't fast enough to do this. It's not usual that somebody looks at the PR a minute after a new push. I am sorry I pushed it too early.